src/common/models/xuiNode.class.ts

```diff
@@ -2,13 +2,13 @@ import { EventEmitter } from 'events'
 import { NextFunction, Request, RequestHandler, Response, Router } from 'express'
 import { XuiNodeOptions } from './xuiNodeOptions.interface'
 import { hasKey, getLogger, XuiLogger } from '../util'
-import { AUTH } from '../../auth/auth.constants' // NOTE: do not shorten this path, tests fail (circular import somewhere)
```

Comment:
Need to check if there are any side effects from importing this - otherwise why was it imported?

Reply:
There is no side affect seen.  Please note 'AUTH' is defined but never used.